src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+
+import duke.errors.DukeException;
+import duke.main.Storage;
+import duke.main.Ui;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Represents a command to add deadlines. Command contains the description and when task is due.
+ */
+public class AddDeadlineCommand extends Command {
+    private String description;
+    private LocalDateTime by;
+
+    /**
+     * Constructor for AddDeadlineCommand
```

Comment:
Should there be an empty line between the first line description and the parameter section?